index/20210312.1.yml

```diff
@@ -0,0 +1,136 @@
+title: Hackathon Nayarit 2021
+lang: es
+title_es: 
+start_utc: 2021-03-12
+end_utc: 2021-03-14
+announcements:
+  - https://www.facebook.com/EduNayarit/posts/2982126628674008
+  - https://twitter.com/Decred_ES/status/1366465819329368066
+  - https://twitter.com/Decred_ES/status/1366465867635183617
+  - https://twitter.com/Decred_ES/status/1365496737566179328
+  - https://matrix.to/#/!clHjlICBEtCtAdTupf:decred.org/$9bNnZG8xlVK9xN-k-0IS2opKchaasmuMISGX7xV8l38
+location: Internet / Facebook { others? }
+organizers:
+  - org: Ministry of Education of Nayarit
+    url: https://www.facebook.com/EduNayarit/
+    person: 
```

Comment:
Elian

Reply:
@adcadde I added two organizations as "organizers". Optionally, it is possible to add also mark a certain person from the org. Do you mean to add our Elian to "Decred Latam team" on the next line, or is there another Elian at the "Ministry of Education of Nayarit" org?